microsoft-edge/webview2/concepts/user-data-folder.md

```diff
@@ -78,16 +78,11 @@ The UDF is created on startup of the WebView2 host app, if the UDF doesn't exist
 <!-- ====================================================================== -->
 ## How many UDFs are created?
 
-Each instance of a WebView2 control is associated with a user data folder (UDF).
+Each instance of a WebView2 control is associated with a WebView2 session.
 
-Each WebView2 session must have a UDF.  There's only 1 active UDF per WebView2 session.
-
-There is at least one UDF per app WebView2 session.  It's possible for your host app to overlap them, by specifying a custom UDF location.
-Or, you can have one UDF per machine.  This depends on how your host app configs the UDF.
-
-A UDF can be either per user, if the app was installed per-user.
-If the host app is installed per-user, each UDF is unique to a user, if not otherwise specified.
+Each WebView2 session has exactly one UDF and a UDF can only have at most one WebView2 session at a time.
```

Comment:
```suggestion
* Each WebView2 session has exactly one UDF.
* A UDF can only have at most one WebView2 session at a time.
```
trying a list, to help digest the two distinct, related points.  not sure how this format flows with the subseq. paragraph's points.
At minimum, insert comma after "one UDF".